examples/simple_zonal_with_acm/acm.tf

```diff
@@ -28,7 +28,7 @@ module "acm" {
 
   secret_type = "ssh"
 
-  policy_bundles = ["https://github.com/GoogleCloudPlatform/acm-policy-controller-library/bundles/policy-essentials-v2022?ref=59f4695394285078f7c2029ec7d0f9ed1d6d700a"]
+  policy_bundles = ["https://github.com/GoogleCloudPlatform/acm-policy-controller-library/bundles/policy-essentials-v2022?ref=dd3f932eefa4c4c44c548144be1c2331d8594689"]
```

Comment:
Was there a reason for this change?

Reply:
The pipeline was failing - https://console.cloud.google.com/cloud-build/builds;region=global/b022a62b-5437-4951-9733-e491b2a82248?project=cloud-foundation-cicd

unknown field "namespace" in 
sh.gatekeeper.constraints.v1beta1.K8sRestrictRoleBindings.spec.parameters.allowedSubjects];